locale/fi/LC_MESSAGES/django.po

```diff
@@ -4443,7 +4478,7 @@ msgstr "Aloita uudelleenarviointi"
 
 #: exercise/templates/exercise/staff/_resubmit_modal.html
 msgid "RE-SUBMIT_TO_SERVICE"
-msgstr "Lähetä arvosteltavaksi uudelleen"
+msgstr "Lähetä arvosteltavaksi uudelleen"#: exercise/templates/exercise/_user_results.html
```

Comment:
Why is the text `#: exercise/templates/exercise/_user_results.html` after the message string?